src/com/capgemini/service/ReservationService.java

```diff
@@ -67,5 +68,13 @@ public ReservationService() {
                 allReservedRooms.addAll(reservation.getRooms());
         }
         return allReservedRooms;
+
+    }
+
+    private void SoftDelete(int reservationID){
```

Comment:
parameter Reservation reservation lijkt me een betere parameter, in plaats van ID meegeven